src/Parser/DemonHunter/Havoc/Normalizers/EyeBeam.js

```diff
@@ -0,0 +1,39 @@
+import SPELLS from 'common/SPELLS';
+
+import EventsNormalizer from 'Parser/Core/EventsNormalizer';
+
+class EyeBeam extends EventsNormalizer {
+	/**
+	* The applybuff from demoic is logges before the cast of Eye Beam. This makes it harder to deal with so reorder it.
```

Comment:
A couple typos here, `demoic` and `logges`

Also I think it's very important to be as clear as possible about what the Normalizer is going to do, in this case something like "This normalizes events so that the Eye Beam applybuff always comes _before_ the Meta Havoc buff" (assuming what I just wrote is a correct summary of what this does)